Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -260,10 +260,13 @@ public class MapboxNavigationService: NSObject, NavigationService {
         
         eventsManager.sendRouteRetrievalEvent()
         router.delegate = self
+        
+        if let routeController = router as? RouteController {
+            routeController.navigator.resume()
```

Comment:
Shouldn't this be a responsibility of an end user? As @S2Ler mentioned pause and resume operation should be performed only via `BillingHandler`, not by accessing `RouteController` directly. This means that we'll need to have `sessionUUID` and there is no ability to know it because `PassiveLocationManager` is created ad-hoc by and user.